app/lib/activity_notifier.rb

```diff
@@ -56,6 +60,13 @@ def each_member(ids, &block)
         .or(member.loan_summaries.includes(:renewal_requests).checked_out)
         .includes(item: :borrow_policy)
       yield member, summaries
+    rescue => e
```

Comment:
I think this is a totally reasonable way to handle failures at this point. Bonus points for sending the exception to AppSignal so we can track them.